CONTRIBUTING.md

```diff
@@ -131,22 +133,34 @@ The following data **must** be present:
   - We ask that you keep it to 4 tags maximum
 - A `license` to be associated with the post
   - This must match the `id` field for one of the values [in our `license.json` file](./content/data/licenses.json)
-  - If you're not familiar with what these licenses mean, view the `explainLink` for each of them in the `license.json` file. It'll help you understand what permissions the public has to the post 
+  - If you're not familiar with what these licenses mean, view the `explainLink` for each of them in the `license.json` file. It'll help you understand what permissions the public has to the post
     - For example, can they modify the article and re-release it or not?
 
 ## Editing a Blog Post
 
 Our blog posts can all be found under [`/content/blog`](./content/blog). Simply find the article based on the URL path and edit the `index.md` file. We'll have one of our editors review the post changes, and we'll try to reach out to the author for them to review your PR as well.
 
-# Translations
+## Translating a Blog Post
+
+If you are adding a translation, make sure to create an [Author Data File](#author-data-file) with the `"translator"` role so that you are credited for your work on the site!
+
+### Finding a Language Code
+
+For any language to be translated, it must have a name and identifier defined in the [`/content/data/languages.json`](./content/data/languages.json) file. If the language is already defined there, simply use its identifier in the following sections; if not, we will need to add it to the file.
+
+Each language code should consist of two lowercase letters. If it includes a region, append a hyphen followed by two more lowercase letters. For example, the code for French is `fr` - to specifically refer to the French dialect in Canada, the code would be `fr-ca`.
+
```

Comment:
```suggestion

> Please use `-` for language region ISO formats, instead of `_`. Instead of `fr_ca`, it'd be `fr-ca`.

```

Adding this since otherwise our code will break